src/assets/js/nav.js

```diff
@@ -63,15 +77,44 @@ dropdownElements.forEach(element => {
             event.preventDefault();
 
             element.classList.toggle("cs-active");
-            ariaExpanded(element);
+            // adjust aria-expanded attribute on the dropdown button only
+            if (dropdownButton) {
+                ariaExpanded(dropdownButton);
+            }
         };
 
         // Pressing Escape will remove the active class from the dropdown. The stopPropagation above will stop the hamburger menu from closing
         if (event.key === "Escape") {
-            element.classList.remove("cs-active");
```

Comment:
Actually, it should be kept as .remove("cs-active") instead of .toggle(), as it was before.
Otherwise, pressing Esc can open the menu too. 

Geoffrey